api/python/query.md

```diff
@@ -49,51 +51,51 @@ A correct response will always look like this:
 
 The response contains an array of dictionaries, each corresponding to one node matching the conditions of the query. Each dictionary's keys match the variables in the query SELECT clause, and the values in the dictionaries are those associated to the given node's query-specified properties.
 
-## Examples and error returns
+## Examples and error responses
```

Comment:
TBH, this is almost a complete copy of the Colab notebook, and I would have just completely removed all this altogether. But since this is v1 I wanted to put minimal effort into it. Maybe in another PR?

Reply:
sg!